_content/gateways/kerlink/config.md

```diff
@@ -9,38 +9,58 @@ zindex: 800
 > 
 > If you have access to the Kerlink wiki it is recommended to follow the firmware upgrade and packet forwarder installation steps from there. This will ensure you are running the latest and most reliable version.
 
-## Configuring packet forwarder
+## Packet forwarders
 
-To configure the gateway, you need to download either [The Things Network's packet forwarder (EU version)](https://github.com/TheThingsNetwork/kerlink-station-firmware/blob/master/dota/dota_thethingsnetwork_v1.3_EU.tar.gz) and [produsb.zip](https://github.com/TheThingsNetwork/kerlink-station-firmware/blob/master/dota/produsb.zip) from our Github.
+Multiple packet forwarders are supported by the Kerlink gateway to forward data to TTN:
+* [Kerlink Semtech Packet Forwarder (SPF)](https://raw.githubusercontent.com/TheThingsNetwork/kerlink-station-firmware/master/dota/dota_spf_3.1.0-klk16_4.1.3-klk8_wirnet_ttn.tar.gz) - uses UDP to forward data to a single LoRaWAN network server
+* [Poly Packet Forwarder](https://github.com/TheThingsNetwork/kerlink-station-firmware/blob/master/dota/dota_thethingsnetwork_v1.3_EU.tar.gz) - can forward to multiple LoRaWAN network servers using UDP

```

Comment:
Shall we stop mentioning this one?

Reply:
Seeing as this doc used Poly as the default one until now, maybe better to leave the link here for reference.